modules/Mail/Spamgourmet/Page.pm

```diff
@@ -1,94 +1,99 @@
 package Mail::Spamgourmet::Page;
 use strict;
 
-my %_templates;  # hash to cache templates
-my %_templatedates; # hash to track template dates (and refresh the cache if they've been updated)
-my $_defaultcontenttype = 'text/html';
+my %_templates;    # hash to cache templates
+my %_templatedates
+    ; # hash to track template dates (and refresh the cache if they've been updated)
```

Comment:
The reflowed line now has a comment AFTER the variable. We might want to avoid this ugliness by moving the comment to the line before the var declaration,